tests/Morphir/Elm/FrontendTests.elm

```diff
@@ -95,16 +97,16 @@ type Cee = Cee
                                 Dict.fromList
                                     [ ( [ "bar" ]
                                       , public
-                                            (Type.typeAliasDefinition []
-                                                (Type.reference (fQName packageName [ [ "a" ] ] [ "foo" ]) [] ())
+                                            (Type.TypeAliasDefinition []
+                                                (Type.Reference () (fQName packageName [ [ "a" ] ] [ "foo" ]) [])
                                             )
                                       )
                                     , ( [ "foo" ]
                                       , public
-                                            (Type.customTypeDefinition []
+                                            (Type.CustomTypeDefinition []
                                                 (public
                                                     [ ( [ "foo" ]
```

Comment:
I think we should change the `Constructor` type to be a custom type in the IR instead of an alias. I think that makes it easier to read the tests.

Reply:
I could have used that here even now that it's an alias. Elm creates an implicit constructor for record type aliases. Let me change this code to use that.

Reply:
Sorry, I confused myself, I thought it was a record. Let me do as you suggest.